samples/confluent-schema-registry-integration/confluent-schema-registry-integration-producer1/pom.xml

```diff
@@ -15,24 +15,20 @@
 	</parent>
 
     <dependencies>
-        <dependency>
```

Comment:
I noticed we have actuator in all the samples by default. I know there are some SCS things in there, but should we keep the pom light and focused on what we are demoing? Or is there some value in getting our SCS actuator code to execute during startup to possible show any issues w/ specific configurations/samples?